paddle/phi/api/yaml/ops.yaml

```diff
@@ -1128,6 +1128,14 @@
   data_transform :
     skip_transform : out_size, size_tensor, scale_tensor
 
+- op : nextafter
+  args : (Tensor x, Tensor y)
+  output : Tensor(out)
+  infer_meta :
+    func : NextafterInferMeta
```

Comment:
reuse `ElementwiseInferMeta` ?

Reply:
Do you mean add nextafter into Elementwise? like elementwise_add ?

Reply:
you mean change to elementwise_nextafter ? and add NextafterRawFunctor? move nextafter to elementwise_functor.h and other file?

Reply:
2023-04-21 17:17:52 ======================================================================
2023-04-21 17:17:52 FAIL: test_static_api (test_nextafter_op.TestNextafterAPI)
2023-04-21 17:17:52 ----------------------------------------------------------------------
2023-04-21 17:17:52 Traceback (most recent call last):
2023-04-21 17:17:52   File "C:\home\workspace\Paddle\build\python\paddle\fluid\tests\unittests\test_nextafter_op.py", line 47, in test_static_api
2023-04-21 17:17:52     out = paddle.nextafter(x, y)
2023-04-21 17:17:52   File "C:\home\workspace\Paddle\build\python\paddle\tensor\math.py", line 5517, in nextafter
2023-04-21 17:17:52     helper.append_op(
2023-04-21 17:17:52   File "C:\home\workspace\Paddle\build\python\paddle\fluid\layer_helper.py", line 45, in append_op
2023-04-21 17:17:52     return self.main_program.current_block().append_op(*args, **kwargs)
2023-04-21 17:17:52   File "C:\home\workspace\Paddle\build\python\paddle\fluid\framework.py", line 4061, in append_op
2023-04-21 17:17:52     op = Operator(
2023-04-21 17:17:52   File "C:\home\workspace\Paddle\build\python\paddle\fluid\framework.py", line 2883, in __init__
2023-04-21 17:17:52     assert (
2023-04-21 17:17:52 AssertionError: Input x not found

Reply:
@jeff41404  if reuse ElementwiseInferMeta and del NextafterInferMeta in binary.cc . the error as this. I don't know why.